docs/reference/troubleshooting/exemptions-and-package-updates.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Exemption and Package Updates (5m)
+
+sidebar:
+  order: 1
+---
+
+
+## Exemptions and Package Updates in the Cluster
+
+This guide provides steps to debug issues with `Exemptions` and `Packages` not being applied or updated in your Kubernetes cluster. Common symptoms include:
+- Changes to Exemptions or Packages are not reflected in the cluster.
+- Expected behavior in workloads remains unaffected.
+- Logs indicate potential Kubernetes Watch failures.
+
+Follow this guide to identify and resolve these issues.
+
+## Initial Checklist
+
+Before diving into detailed debugging, ensure the following:
+
+- **Verify Configuration**:
+   - Ensure that Exemptions and Packages are defined correctly in your manifests.
+   - Refer to the specification documents for correct schema and examples:
+     - [Packages Specification](https://github.com/defenseunicorns/uds-core/blob/main/docs/reference/configuration/custom%20resources/packages-v1alpha1-cr.md)
+     - [Exemptions Specification](https://github.com/defenseunicorns/uds-core/blob/main/docs/reference/configuration/custom%20resources/exemptions-v1alpha1-cr.md)
+
+- **Namespace for Exemptions**:
+   - Ensure Exemptions are applied in the `uds-policy-exemptions` namespace.
+
+- **Cluster and Deployment Status**:
+   - Confirm the cluster and relevant controller deployments are running without errors: 
+      ```bash
+        kubectl get pods -n pepr-system
+      ```
+
+## Troubleshooting Kubernetes Watch
+
+Kubernetes Watch is a mechanism used to monitor resource changes in real-time. Failures in Watch can cause Exemptions and Package updates to not propagate.
+
+### Steps to Check Watch Logs
+
+1. **Identify the Controller Pod**:
+   - Check the logs of the controller managing Exemptions using the following command:
+     ```bash
+     kubectl logs -n pepr-system -l app -f | egrep "Processing exemption"
```

Comment:
I'm not entirely sure why but this is not working for me. If I put the exact pod name into the `kubectl logs` command that works out, but I can't seem to get it to work with any label selectors?

Reply:
```
kubectl logs -n pepr-system deploy/pepr-uds-core --all-pods | grep "Processing exemption"
```
This works better for me - not sure if we need the `--all-pods`.

Reply:
ahah yeah i can't believe I added all-pods 😆 , i wrote that feature into k8s and I was just noticing it was out, must had been subconscious but a definite oversight. Will get this fixed